lively.morphic/text/morph.js

```diff
@@ -3341,15 +3340,21 @@ export class Text extends Morph {
 
   onHoverIn (evt) {
     super.onHoverIn(evt);
-    this.scrollActive = true;
-    this.makeDirty();
+    // Only necessary for morphs which have a document since we need to take action here, due to our decoupled scroll implementation.
+    if (this.document) {
+      this.scrollActive = true;
+      this.makeDirty();
+    }
   }
 
   onHoverOut (evt) {
     super.onHoverOut(evt);
     if (touchInputDevice) return;
-    this.scrollActive = false;
-    this.makeDirty();
+    // Only necessary for morphs which have a document since we need to take action here, due to our decoupled scroll implementation.
+    if (this.document) {
+      this.scrollActive = false;
+      this.makeDirty();
+    }
```

Comment:
```suggestion
   this.ensureScrollLayer()
```

Reply:
This does not cut it, as the values for `scrollActive` are changing. But I will extract the method.